coderabbit.yaml

```
@@ -1,47 +1,114 @@
-settings:
-  language: go
-  go_version: "1.24"
-  enable_auto_review: true
-  enable_auto_merge: false
-  require_tests: true
-  require_documentation: true
+  yamllint:
+    enabled: true
+
+# Path instructions for Go 1.24 compatibility
+reviews:
```

YAMLlint (1.37.1)

[error] duplication of key "reviews" in mapping

(key-duplicates)